lib/cypress/patient_zipper.rb

```diff
@@ -40,20 +38,18 @@ def initialize(measures, start_time, end_time)
         @valueset_json_map[bundle.id] = value_sets.to_json
       end
       @measures_json = @measures.to_json
+      CAT_EXPORTER.load_measures_and_value_sets(@measures_json, @valueset_json_map[@measures.first.bundle_id])
```

Comment:
@valueset_json_map is probably no longer needed.  can pass in HealthDataStandards::SVS::ValueSet.where(:oid.in => @measures.map(&:oids).flatten, bundle_id: @measures.first.bundle_id).to_json directly